### PR TITLE
update troubleshoot-np.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2392 
added info about whitelisting websites

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ Set-MpPreference -EnableNetworkProtection Enabled
 
 If you've tested the feature with the demo site and with audit mode, and network protection is working on pre-configured scenarios, but is not working as expected for a specific connection, use the [Windows Defender Security Intelligence web-based submission form](https://www.microsoft.com/en-us/wdsi/filesubmission) to report a false negative or false positive for network protection. With an E5 subscription, you can also [provide a link to any associated alert](../microsoft-defender-atp/alerts-queue.md).
 
+## Exclude website from Network Protection scope
+
+To whitelist the website that is being blocked (false positive), add its URL to the [list of trusted sites](https://blogs.msdn.microsoft.com/asiatech/2014/08/19/how-to-add-web-sites-to-trusted-sites-via-gpo-from-dc-installed-ie10-or-higher-ie-version/). Web resources from this list bypass the Network Protection check.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md]:
```suggestion
To whitelist the website that is being blocked (false positive), add its URL to the [list of trusted sites](https://blogs.msdn.microsoft.com/asiatech/2014/08/19/how-to-add-web-sites-to-trusted-sites-via-gpo-from-dc-installed-ie10-or-higher-ie-version/). Web resources from this list bypass the network protection check.
```

File: windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ Set-MpPreference -EnableNetworkProtection Enabled
 
 If you've tested the feature with the demo site and with audit mode, and network protection is working on pre-configured scenarios, but is not working as expected for a specific connection, use the [Windows Defender Security Intelligence web-based submission form](https://www.microsoft.com/en-us/wdsi/filesubmission) to report a false negative or false positive for network protection. With an E5 subscription, you can also [provide a link to any associated alert](../microsoft-defender-atp/alerts-queue.md).
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md]:
```suggestion
If you've tested the feature with the demo site and with audit mode, and network protection is working on pre-configured scenarios, but is not working as expected for a specific connection, use the [Windows Defender Security Intelligence web-based submission form](https://www.microsoft.com/wdsi/filesubmission) to report a false negative or false positive for network protection. With an E5 subscription, you can also [provide a link to any associated alert](../microsoft-defender-atp/alerts-queue.md).
```

Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md]:
```suggestion
If you've tested the feature with the demo site and with audit mode, and network protection is working on pre-configured scenarios, but is not working as expected for a specific connection, use the [Windows Defender Security Intelligence web-based submission form](https://www.microsoft.com/wdsi/filesubmission) to report a false negative or false positive for network protection. With an E5 subscription, you can also [provide a link to any associated alert](../microsoft-defender-atp/alerts-queue.md).
```

File: windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md [hunk 3]
```diff
@@ -67,6 +67,10 @@ Set-MpPreference -EnableNetworkProtection Enabled
...
 
+## Exclude website from Network Protection scope
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/troubleshoot-np.md]:
```suggestion
## Exclude website from network protection scope
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix , @JohanFreelancer9 ,
your recommendations were applied, thank you!

**Comment by mypil** [comment]
@Justinha - Can you please assist to check, approve and merge the content of this PR based on issue #2392?

Thank you.

**Comment by mypil** [comment]
@Justinha @Dansimp @AndreaBarr - Can you please assist to check, approve and merge the content of this PR based on issue #2392?

Thank you.

**Pull request merged**